src/styles/components/framework-configuration/styles.less

```diff
@@ -7,3 +7,14 @@
 .pre-install-notes.message.message-warning p {
   display: inline;
 }
+
+.framework-form-yaml-editor {
+  // todo use same border color as the inputs
```

Comment:
`@form-control-border-color` might be what you are looking for.

Reply:
Nice, thanks 👍